templating.rst

```diff
@@ -405,10 +404,9 @@ for several types of templates, each which lives in a specific location:
   is missing (e.g. ``Blog``), the template lives at
   ``Resources/views/layout.html.twig`` inside AcmeBlogBundle.
 
-In the :doc:`/templating/overriding` section, you'll find out how each
-template living inside the AcmeBlogBundle, for example, can be overridden
-by placing a template of the same name in the ``templates/bundles/AcmeBlogBundle/``
-directory. This gives the power to override templates from any vendor bundle.
+Using this namespaced syntax instead of their real file paths to refer to bundle
+templates is part of the bundle overriding mechanism. This allows applications
```

Comment:
This part is not exact, but do we care? `bundle overriding mechanism` => `Twig overriding mechanism`

Reply:
You are right! I've reworded it.